docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -999,17 +1044,17 @@ Note that the generated CA is only valid for development purposes and can only b
 
 . Generate a development CA:
```

Comment:
```suggestion
To generate a development CA:
```

Reply:
Also, line 1040 (the heading) should probably be `=== Generating a development CA`